content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
 
 ```bash
 cd my-app
 cd src
 
-# If you're using a Mac or Linux:
+# यदि आप एक मैक या लिनक्स का उपयोग कर रहे हैं:
 rm -f *
 
-# Or, if you're on Windows:
+# या, यदि आप विंडोज पर हैं:
 del *
 
-# Then, switch back to the project folder
+# उसके बाद, प्रोजेक्ट फ़ोल्डर पर वापस जाएँ
 cd ..
 ```
 
-4. Add a file named `index.css` in the `src/` folder with [this CSS code](https://codepen.io/gaearon/pen/oWWQNa?editors=0100).
+4. [इस CSS कोड](https://codepen.io/gaearon/pen/oWWQNa?editors=0100) के साथ `src/` फ़ोल्डर में `index.css` नामक फ़ाइल बनायें।
 
-5. Add a file named `index.js` in the `src/` folder with [this JS code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010).
+5. [इस JS कोड](https://codepen.io/gaearon/pen/oWWQNa?editors=0010) के साथ `src/` फ़ोल्डर में `index.js` नामक एक फ़ाइल बनायें।
 
-6. Add these three lines to the top of `index.js` in the `src/` folder:
+6. `src/` फ़ोल्डर में `index.js` के शीर्ष पर इन तीन पंक्तियों को जोड़ें:
 
 ```js
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 ```
 
-Now if you run `npm start` in the project folder and open `http://localhost:3000` in the browser, you should see an empty tic-tac-toe field.
+अब अगर आप प्रोजेक्ट फोल्डर में `npm start` चलाते हैं और ब्राउज़र में `http://localhost:3000` ओपन करते हैं, तो आपको एक खाली टिक-टैक-टो फ़ील्ड दिखना चाहिए।
 
-We recommend following [these instructions](https://babeljs.io/docs/editors/) to configure syntax highlighting for your editor.
+हम आपके संपादक के लिए वाक्य रचना हाइलाइटिंग को कॉन्फ़िगर करने के लिए [इन निर्देशों](https://babeljs.io/docs/editors/) का पालन करने की सलाह देते हैं।
 
 </details>
 
-### Help, I'm Stuck! {#help-im-stuck}
+### मदद, मैं अटक गया हूँ! {#help-im-stuck}
 
-If you get stuck, check out the [community support resources](/community/support.html). In particular, [Reactiflux Chat](https://discord.gg/0ZcbPKXt5bZjGY5n) is a great way to get help quickly. If you don't receive an answer, or if you remain stuck, please file an issue, and we'll help you out.
+यदि आप अटक जाते हैं, तो [सामुदायिक सहायता संसाधनों](/community/support.html) की जांच करें। विशेष रूप से, [Reactiflux चैट](https://discord.gg/0ZcbPKXt5bZjGY5n) जल्दी से सहायता प्राप्त करने का एक शानदार तरीका है। यदि आपको कोई उत्तर नहीं मिलता है, या यदि आप अटके रहते हैं, तो कृपया एक समस्या दर्ज करें, और हम आपकी मदद करेंगे।
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-Now that you're set up, let's get an overview of React!
+अब जब आप सेट हो गए हैं, तो React का ओवरव्यू करें!
 
-### What Is React? {#what-is-react}
+### React क्या है? {#what-is-react}
 
-React is a declarative, efficient, and flexible JavaScript library for building user interfaces. It lets you compose complex UIs from small and isolated pieces of code called "components".
+उपयोगकर्ता इंटरफेस के निर्माण के लिए React एक घोषणापत्र, कुशल और लचीली जावास्क्रिप्ट लाइब्रेरी है। यह आपको "कौम्पोनॅन्ट्स" नामक कोड के छोटे और पृथक टुकड़ों से जटिल यूआई की रचना करने देता है।
```

Comment:
वर्णनात्मक is better translation for declarative

Reply:
flexible can be left as is maybe?

Reply:
we were not translating UI  at all if I remember correctly

Reply:
देता -> deti

Reply:
> we were not translating UI at all if I remember correctly

Yes, we are not. I guess I missed few of them